frontend/.babelrc

```diff
@@ -1,34 +1,72 @@
 {
   "presets": [
-    "env",
-    "react"
+    [
+      "@babel/preset-env",
+      {
+        "modules": false
+      }
+    ],
+    "@babel/preset-react"
   ],
   "plugins": [
-    "transform-class-properties",
-    "transform-object-rest-spread",
+    "lodash",
+    "@babel/plugin-proposal-class-properties",
+    [
+      "@babel/plugin-proposal-object-rest-spread",
+      {
+        "loose": true
+      }
+    ],
+    [
+      "@babel/plugin-transform-spread",
+      {
+        "loose": true
+      }
+    ],
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-export-default-from",
     "transform-export-extensions",
-    "transform-react-jsx"
+    [
+      "@babel/plugin-syntax-dynamic-import",
+      {
+        "loose": true
+      }
+    ],
+    [
+      "module-resolver",
+      {
+        "cwd": "packagejson",
+        "alias": {
```

Comment:
are these alias needed?

Reply:
ah good catch. They are not needed. updated.